app/src/main/java/org/mozilla/tv/firefox/utils/UrlUtils.kt

```diff
@@ -158,8 +158,11 @@ object UrlUtils {
      */
     @JvmStatic
     fun toUrlBarDisplay(url: String): String {
-        return when (url) {
-            URLs.APP_URL_HOME -> "" // Empty string forces the EditText to show hint text
+        return when {
+            url == URLs.APP_URL_HOME -> "" // Empty string forces the EditText to show hint text
+            // GeckoView loads [data: url] when RequestInterceptor.InterceptionResponse.Content()
+            // due to lack of internal page api (bug #1535131)
+            url.startsWith("data:", false) -> ""
```

Comment:
Hm, it looks like from bug 1535131, this means that the url will actually just show the hint, but not any sort of "firefox:about" url? I guess that's fine for this workaround.
From the discussion though, it looks like this isn't high enough priority for GeckoView to fix because it'll be a non-trivial amount of work, so our "about:" pages in GV Firefox will just show the hint - heads up @athomasmoz is that fine?

Reply:
Agreed. This sounds fine to me